src/api.coffee

```diff
@@ -16,7 +17,7 @@ privateAppEnvVars = [
 
 module.exports = (application) ->
 	api = express()
-	api.use(bodyParser())
```

Comment:
(Done)
Did not use a namespace as endpoints that use parsed bodies are in several different actions, including images and containers.